server/src/main/java/kr/codesquad/library/LibraryApplication.java

```diff
@@ -0,0 +1,13 @@
+package kr.codesquad.library;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+
+@SpringBootApplication
+public class LibraryApplication {
+
+	public static void main(String[] args) {
+		SpringApplication.run(LibraryApplication.class, args);
+	}
+
+}
```

Comment:
스프링에서 생성해준 기본 Application 이 클래스의 마지막에 개행을 한 형태로 생성되는 것은 알고 있지만, 일반적인 클래스의 처음은 개행을 하고, 마지막은 개행을 하지 않는 것으로 알아요.

```suggestion
@SpringBootApplication
public class LibraryApplication {

	public static void main(String[] args) {
		SpringApplication.run(LibraryApplication.class, args);
	}
}
```

Reply:
엔터키 잘못누른듯 인식을 못했네요 개행없애버려야겠습니다